### DISCUSSION TITLE
Signups not working anymore for me

### DISCUSSION THREAD

**Discussion by Cesar-Paredes** [supabase/supabase#29672] [Questions] [answered]
My signups were working fine yesterday and today I'm getting this error:
eEmail address \"arrivederchiiiiman@gmail.com\" cannot be used as it is not authorized"

I have tried several emails that are not already in my authentication table, I have tried secured passwords using letters, numbers and symbols and it is not working.
Does anyone else getting this error? is this somehow related to the technical issues supabase was having today?

Note that logins are working fine, its only the signups that are not working anymore for me.

![image](https://github.com/user-attachments/assets/c5a77de4-9327-4641-a985-94f0d3eaad71)

**Comment by GaryAustin1** [comment]
When using the built in SMTP  provider all emails need to be part of your organization.
You should see a banner a the top of your dashboard...
![image](https://github.com/user-attachments/assets/ea73be39-223a-4ef8-be9e-f10a2dc3f115)

**Reply by Cesar-Paredes** [reply]
I saw that as well but I am not sending emails to my users from my authentication table, look at the screenshot below:
In my providers I don't send any type of email to my users, so the "Auth emails are restricted to your organization's members" do not apply to me.
![image](https://github.com/user-attachments/assets/681fc3d0-7bd8-4722-b73f-0ac78197770a)

I debuged further my application and I found more information about the error. Again this error started to happen today but yesterday the signups were working fine.

I dont understand why any email is not authorized to signup, my policies are public.

{
  __isAuthError: true,
  name: "AuthApiError",
  status: 400,
  code: "email_address_not_authorized",
}

**Reply by GaryAustin1** [reply, marked as answer]
It does not matter if you send the email or not currently.  The Learn more link has the details.
![image](https://github.com/user-attachments/assets/9df181d6-299a-485e-9394-0aaa254132b3)
https://github.com/orgs/supabase/discussions/29370

**Reply by Cesar-Paredes** [reply]
ok I see. Thanks for the info and taking the time!

Supabase is making this a bit tedious when we are not even trying to use email confirmation, we just want to authenticate but we still have to set up the STMP.

**Reply by RodRe1ss** [reply]
hi, is there a solution because im just starting to use supabase and im very confused

**Reply by GaryAustin1** [reply]
@RodRe1ss 
If you use the built in SMTP email service your are limited to your dashboard signup email (or dashboard teammates you invite) for app signups.  It is just for kicking the tires.

If you are doing more advanced testing and certainly production you need to get your own SMTP provider.

There are some other more advanced cases that are being worked thru the past week because of a change but by the time (and if) they impact you should all be sorted.

**Reply by darrenpaul-mira** [reply]
I had the same issue, disabling facebook sign up seemed to fixed this problem for me 😕
